.gitignore

```diff
@@ -2,3 +2,4 @@
 .php_cs.cache
 composer.lock
 phpunit.xml
+/.idea/
```

Comment:
Please remove this. Instead, use a global gitignore, and exclude this directory for all your git repo : https://github.com/greg0ire/dotfiles/blob/master/gitconfig#L33